File: Lenguaje/Codeforces/Div 4/Round 784/A_Division/A_Division.cpp
#include <iostream>
using namespace std;
int t,n;
int main(){
    ios_base::sync_with_stdio(0);cin.tie(0);cout.tie(0);
    cin>>t;
    while(t--){
        cin>>n;
        if(n>=1900)cout<<"Division 1\n";
        else if(n>=1600)cout<<"Division 2\n";
        else if(n>=1400)cout<<"Division 3\n";
        else cout<<"Division 4\n";
    }
    return 0;
}

File: Lenguaje/Codeforces/Div 4/Round 784/A_Division/A_Division.py
t = int(input())
while(t > 0):
    t=t-1;
    n = int(input())
    if(n>=1900): print("Division 1")
    elif(n>=1600): print("Division 2")
    elif(n>=1400): print("Division 3")
    else: print("Division 4")